apps/frontend/src/components/settings/proposalWorkflow/EmailActionConfig.tsx

```diff
@@ -38,6 +40,101 @@ const EmailActionConfig = ({
   isEmailTemplateRequired = false,
 }: EmailActionConfigProps) => {
   const classes = useStyles();
+  const [otherRecipientsValue, setOtherRecipientsValue] = useState('');
+  const [otherRecipientsFieldError, setOtherRecipientsFieldError] = useState<
+    string | null
+  >(null);
+
+  const getRecipientIndexByName = (
+    recipientName: EmailStatusActionRecipients
+  ) =>
+    recipientsWithEmailTemplate.findIndex(
+      (item) => item.recipient?.name === recipientName
+    );
+
+  const isInList = (email: string, list?: string[] | null) => {
+    return list?.includes(email);
+  };
+
+  const isEmail = (email: string) => {
+    return /[\w\d\.-]+@[\w\d\.-]+\.[\w\d\.-]+/.test(email);
```

Comment:
If I'm correct, it's not necessary when the input type is an email because it already has basic validation. Adding an additional regex like this might reject valid email addresses.

Reply:
I added the Yup validation here instead of custom one now. This is done because on the backend we use Yup validation for email and if I allow user to send data with only html email field validation we end up with backend input validation errors which is not ideal.